src/Microsoft.VisualStudio.Editors/ResourceEditor/Resource.vb

```diff
@@ -1481,11 +1482,11 @@ Namespace Microsoft.VisualStudio.Editors.ResourceEditor
                     'Value is not Nothing
 
                     'Verify that the new value is of the same type as before.
-                    If Not GetValueType().Equals(NewResourceValue.GetType()) Then
+                    If GetValueType() IsNot NewResourceValue.GetType() Then
                         'Exception to this rule.  We need to handle Byte() <-> MemoryStream conversion automatically
-                        If NewResourceValue.GetType().Equals(GetType(Byte())) AndAlso GetValueType().Equals(GetType(MemoryStream)) Then
```

Comment:
These aren't these checks and am very worried about changing 15 year old with no tests.

This will now return true for `MyMemoryStream : MemoryStream`

Reply:
Quite right. This is a R# bug.

https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AbEA3YaAuIUArgHYA+AkgLYAO0+AzgAQDKAno/jNQLABQABWLAMASzDMAshAAmxDDGaCoEAOZQAhn37M9ykeMmsR0zWNIAKAJQD995gBEx1ZiwC8zAERe7DvX7+AMIQpIwQigB0AOpQYtwAMhYwlgAq7LQwAPIAZm7MlCxZwABWMGD4trr+zCFhETAxcYnJloyRAOIw+OmZNpEAogCOxJoYjJZdPRkpxWUV1ouBzAOksmwiAqvrMvKKQA=